docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -189,13 +202,17 @@ If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and
 
 The shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from abruptly decreasing the tenant shard size when shuffle sharding is enabled for ingesters on the read path. This is because when shuffle sharding is disabled, the queriers check all ingesters for blocks, whereas when it's enabled, they only check the ones that are assigned to that tenant through the shuffle-shard.
 
-To safely decrease the tenant shard size, follow these steps.
+To safely decrease the tenant shard size for classic architecture, follow these steps:
 
 1. Disable shuffle sharding on the ingester read path via `-querier.shuffle-sharding-ingesters-enabled=false`.
 1. Decrease the configured tenant shard size.
 1. Wait for at least the amount of time specified via `-blocks-storage.tsdb.retention-period`.
 1. Re-enable shuffle sharding on the ingester read path via `-querier.shuffle-sharding-ingesters-enabled=true`.
 
+{{< admonition type="note" >}}
```

Comment:
We've already specific that this section is for classic architecture. I think it's overkill to state that again here.